docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  service:
```

Comment:
Im not sure about just a `service` instead of `prefix-service`. I've added `fastrack` prefix before to quickly see and find container just in case and also to avoid collisions with other running containers. what do you think?

Reply:
Docker compose takes care of that for us. It always prefixes the container names with the project name (by default it's the directory containing `docker-compose.yml`).

Reply:
yeeeeep. it just looks ugly and you always have to find =((

Reply:
I disagree. Here is the output of `docker ps` on my machine, where the names are very obvious:
```
CONTAINER ID   IMAGE                              COMMAND                  CREATED          STATUS          PORTS                                                 NAMES
a2e6d53aa022   fasttrackml-service                "fml server"             11 seconds ago   Up 11 seconds                                                         fasttrackml-service-1
8a7d1fb20d3a   postgres:latest                    "docker-entrypoint.s…"   17 seconds ago   Up 17 seconds   5432/tcp                                              fasttrackml-postgres-1
```
(Scroll all the way to the left to see them)

Reply:
I mean when we have a prefix it is always a bit easier to see and group them.

Reply:
ok, no problem.